agenta-backend/agenta_backend/resources/evaluators/evaluators.py

```diff
@@ -2,11 +2,17 @@
     {
         "name": "Exact Match",
         "key": "auto_exact_match",
-        "direct_use": True,
+        "direct_use": False,
```

Comment:
This should still be direct use, just configurable. We don't want to add friction just because we have added this option

Reply:
Exact match requires access to the correct answer and direct_use evaluators are persisted on app launch

Reply:
we now create direct-use evaluators with the default correct_answer_keys
so my change there this is not needed anymore